.gitconfig.khan

```diff
@@ -30,6 +35,9 @@
   outgoing = "!git fetch >/dev/null 2>&1; git log @{upstream}..HEAD"
   incoming = "!git fetch >/dev/null 2>&1; git log HEAD..@{upstream}"
   update-submodule = !bash -c 'git pull --no-rebase && pushd $1 && git co master && git pull && popd && git commit ${1%/} -m Substate\\ for\\ ${1%/} -m Test\\ plan: -m None && git push' --
+  pup = "!git push origin $(git branch-name)"
+  pown = "!git pull origin $(git branch-name)"
+  pupl = pup --force-with-lease
```

Comment:
We could bike-shed these names all day.  I wonder if you want to do a poll and see what folks like the best.  My own personal aliases for these are `rpush` and `rpull` (`r` for `remote`) and `fpush` (for `force`).

Reply:
I'm fine with whatever you think is best. There's also nothing really stopping us from having more than one choice and letting folks chose. And obviously, people can create whatever aliases they want.

I personally like `pup` and `pown` because it's less characters to type and because the letter combo `r` + `p` is super uncommon in English, so takes longer to think about and type. (and `fp` too) 🤷‍♀️

Reply:
> because the letter combo r + p is super uncommon in English

Which just makes the tab completion work better!  :-)

My suggestion was to do a poll in slack.  But as you say we can have more than one.  I'm fine with whatever.